src/app/features/registration/registration.component.html

```diff
@@ -1 +1,49 @@
-<p>registration works!</p>
+<div class="app-registration">
+    <form [formGroup]="userForm" (ngSubmit)="onSubmit()" class="text-center border border-light p-5">
+        <p class="h4 mb-4">Registration</p>
+        <div class="form-group">
+            <label for="name">Name</label>
+            <div class="form-row mb-4">
+                <div class="col">
+                    <input type="text" formControlName="name" id="formName" class="form-control" placeholder="name"
+                        required [ngClass]="{ 'is-invalid': submitted && name?.errors }" />
+                    <div *ngIf="submitted && name?.errors" class="invalid-feedback">
+                        <div *ngIf="name?.errors?.['required']">
+                            Name is required
+                        </div>
+                        <div *ngIf="name?.errors?.['minLength']">
+                            Name must be at least 6 characters long
+                        </div>
+                    </div>
```

Comment:
It looks fine, but I just want to show some more options here:
- using a method that extracts error text:
```suggestion
                    <div *ngIf="submitted && name?.errors" class="invalid-feedback">
                        {{ getErrorMessage() }}
                    </div>
```
- using a custom pipe that gets error text:
```suggestion
                    <div *ngIf="submitted && name?.errors" class="invalid-feedback">
                        {{ name?.errors | registrationFormError }}
                    </div>
```